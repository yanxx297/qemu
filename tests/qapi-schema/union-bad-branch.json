# we reject normal unions where branches would collide in C
{ 'type': 'One',
  'data': { 'string': 'str' } }
{ 'type': 'Two',
  'data': { 'number': 'int' } }
{ 'union': 'MyUnion',
  'data': { 'one': 'One',
            'ONE': 'Two' } }
