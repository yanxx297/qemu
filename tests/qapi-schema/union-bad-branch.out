[OrderedDict([('type', 'One'), ('data', OrderedDict([('string', 'str')]))]),
 OrderedDict([('type', 'Two'), ('data', OrderedDict([('number', 'int')]))]),
 OrderedDict([('union', 'MyUnion'), ('data', OrderedDict([('one', 'One'), ('ONE', 'Two')]))])]
[{'enum_name': 'MyUnionKind', 'enum_values': None}]
[OrderedDict([('type', 'One'), ('data', OrderedDict([('string', 'str')]))]),
 OrderedDict([('type', 'Two'), ('data', OrderedDict([('number', 'int')]))])]
